src/components/__snapshots__/StatedBalanceForm.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`StatedBalanceForm renders without crashing given the required props 1`] = `ShallowWrapper {}`;
+exports[`StatedBalanceForm renders without crashing given the required props 1`] = `ReactWrapper {}`;
```

Comment:
those snapshots are strange xDD
what is their purpose?

Reply:
If I understand correctly it checks if given props are used in a good context and if it renders without problems. What's wrong using it?

Reply:
I think it is ok, just we have other tests which would fail if it would not render, am I rite?
I hoped at least that it checks that component renders the same tags...